doc/user/layouts/shortcodes/admonition.html

```diff
@@ -0,0 +1,10 @@
+
+{{- $type := .Get "type" | default "note" -}}
+<div class="admonition">
```

Comment:
What's the intended use of this new shortcode (vs. `Note` and `Warning`)?

Reply:
Nice! Minor preference to name it `annotation` instead.

Reply:
Can update to annotation.